packages/cid-verifier/src/index.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-env serviceworker */
+
+import { Router } from 'itty-router'
+
+import { verificationGet, verificationPost } from './verification.js'
+import { versionGet } from './version.js'
+
+import { addCorsHeaders, withCorsHeaders } from './cors.js'
+import { errorHandler } from './error-handler.js'
+import { envAll } from './env.js'
+
+const router = Router()
+
+// https://developer.mozilla.org/en-US/docs/Web/API/FetchEvent
+/** @typedef {ExecutionContext} Ctx */
+
+router
+  .all('*', envAll)
+  .get('/version', withCorsHeaders(versionGet))
+  .get('/verification', withCorsHeaders(verificationGet))
+  .post('/verification', withCorsHeaders(verificationPost))
```

Comment:
What do you think on having this as just `/`?

Considering we will deploy this as `cid-verifier.dag.haus`, API endpoint with `/verification` seems a bit weird to me. We will (in the future) want a way to read from KV info too. So thinking about:

```
GET /
POST /
GET /metadata (future)
```

(this is future work, not here, just to help with semantics) Metadata would return all the information we know from all the services we already requested info